src/batched/dense/KokkosBatched_Vector.hpp

```diff
@@ -255,6 +255,18 @@ class ArithTraits<KokkosBatched::Vector<KokkosBatched::SIMD<T>, l>> {
     return val;
   }
 
+  static KOKKOS_FORCEINLINE_FUNCTION val_type conj(const val_type &val) {
```

Comment:
This should be defined in `Kokkos::ArithTraits` and `Kokkos::complex` already

Reply:
Here we inject own `Kokkos::ArithTraits` definition for `Vector<SIMD<T>>`, which had no `conj()` implementation.
I added it, because it's needed to handle SIMD vector views inside regular (non-MKL) `SerialGemv` with `Trans::ConjTranspose`.